src/test/resources/stories/Examples.story

```diff
@@ -3,6 +3,7 @@ Scenario: Stock trade alert
 Given a stock of symbol <symbol> and a threshold <threshold>
 When the stock is traded at price <price>
 Then the alert status should be status <status>
+When I have first parameter <symbol> and second parameter <symbol>
```

Comment:
@ikalinin1 No, wait. This exact case you added. With your changes the agent will report only 1 parameter. But it's not true, there are two of them. I wouldn't like hiding any parameters.

Reply:
Sorry, can't get what you mean. My point here that in this particular use case only 1 parameter will be reported. This is not what's expected.

Reply:
I think we have a misunderstanding :) As far as I understand the logic: you are parsing references to examples table parameters e.g. values referenced via `<link>` and then getting the values from the examples table.